respa_admin/templates/respa_admin/common/_periods_accordion.html

```diff
@@ -42,6 +42,10 @@
          role="tabpanel"
          id="collapse{{ id }}"
          aria-labelledby="heading{{ id }}">
+        <input type="hidden"
```

Comment:
Later versions of Django (4.x?) allow you to specify `deletion_widget` for a `Formset` class, so we could just do `{{ form.DELETE }}`.